Gruntfile.js

```diff
@@ -78,6 +79,17 @@ module.exports = function (grunt) {
     return requireOptions;
   }
 
+  var WIDGET_VERSION_TEXT = '<%= widgetversion %>';
+
+  function escapeRegexText(text) {
+    return text.replace(/([.*+?^=!:${}()|\[\]\/\\])/g, '\\$1');
+  }
+
+  function substituteWidgetVersion(content) {
+    var widgetVersionRegex = new RegExp(escapeRegexText(WIDGET_VERSION_TEXT));
```

Comment:
I think we might have talked about this when we were doing it in the authSdk, but why do we need a regex here? Why not just a simple text replace?